src/workers/postUpdated.ts

```diff
@@ -568,8 +568,14 @@ const getSourcePrivacy = async ({
     }
 
     const source = await query.getOne();
-    return source?.private;
+
+    // if source is not found this will return undefined
+    // and DB will default to false, which means post will be public
+    return !!source?.private;
```

Comment:
In case we can't find the source what would we do? Not sure when this would happen but maybe explicitly also return private: false

or change above to be `getOneOrFail` to fallback to catch case below

Reply:
if there's no source we should ignore the message and log it just in case

Reply:
done, will monitor after deploy for any errors on processing